### PR TITLE
make flynt incremental

### PR DESCRIPTION
**Pull request by balopat**
Fixes https://github.com/quantumlib/Cirq/issues/3901.

### CODE REVIEW

File: check/format-incremental [hunk 1]
```diff
@@ -109,6 +105,11 @@ if (( any_files == 0 )); then
     exit 0
 fi
 
+# TODO: Remove '-e cirq/ops/pauli_string.py' once the line is covered (https://github.com/quantumlib/Cirq/issues/3804)
+flynt ${format_files} --quiet -f -e cirq/ops/pauli_string.py
+FLYNTSTATUS=$?
+
+echo "Running black..."
```
Comment (mpharrigan) [inline check/format-incremental:112]: people may not know this is a formatter. sounds kinda funny

Comment (balopat) [reply check/format-incremental:112]: oh, that is so true :D

**Comment by balopat** [comment]
@mpharrigan or @vtomole this is ready for your eyes and LGTMs :)

**Review by vtomole — APPROVED** [review]
Ran locally. Much better output and super-fast!

**Pull request merged**